cpp/src/IceSSL/SChannelTransceiverI.cpp

```diff
@@ -765,8 +762,22 @@ SChannel::TransceiverI::initialize(IceInternal::Buffer& readBuffer, IceInternal:
 
         string trustError;
         PCCERT_CHAIN_CONTEXT certChain;
-        if(!CertGetCertificateChain(_engine->chainEngine(), cert, 0, cert->hCertStore, &chainP,
-                                    CERT_CHAIN_REVOCATION_CHECK_CACHE_ONLY, 0, &certChain))
+        DWORD dwFlags = 0;
```

Comment:
Are we changing the default behavior here? Or does the cache only flag had no effect if the revocation flags weren't set?

Reply:
Yes I think this is only relevant if revocation checks are enabled, this is the same that .NET runtime does

https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/ChainPal.BuildChain.cs#L124-L139